src/ResourceManager/Automation/Commands.Automation/Cmdlet/NewAzureAutomationModule.cs

```diff
@@ -39,6 +39,7 @@ public class NewAzureAutomationModule : AzureAutomationBaseCmdlet
         /// </summary>
         [Parameter(Position = 3, Mandatory = true, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The url to a module zip package.")]
+        [Alias("ContentLinkUri")]
```

Comment:
@vrdmr I think it would be better if we changed the name of this parameter to `ContentLinkUri` and added an alias to `ContentLink`. That way, when users tab through the parameters, `ContentLinkUri` will show up and will be consistent with what is shown when tabbing through the parameters of `Set-AzureRmAutomationModule`.